src/khepri_path.erl

```diff
@@ -310,6 +314,18 @@ from_string([Char] = Rest, [] = ReversedPath)
   when ?IS_SPECIAL_KHEPRI_PATH_COMPONENT(Char) ->
     finalize_path(Rest, ReversedPath);
 
+from_string([$/, $/ | Rest], ReversedPath) ->
+    %% Two consecutive forward slashes mean there is an empty binary
+    %% component.
+    ReversedPath1 = prepend_component(<<>>, ReversedPath),
+    from_string([$/ | Rest], ReversedPath1);
+
+%from_string([$/], ReversedPath) ->
+%    %% A path ending with a forward slash means that the path ends with an
+%    %% empty binary component.
+%    ReversedPath1 = prepend_component(<<>>, ReversedPath),
+%    finalize_path([], ReversedPath1);
```

Comment:
This ends up being covered by the pattern above, right?

Reply:
Indeed, this was left over during development. Removed, thank you!